.github/ISSUE_TEMPLATE/engineering-manager-onboarding.md

```diff
@@ -0,0 +1,67 @@
+---
+name: Engineering Manager Onboarding
+about: A list of things to do as a new manager to become familiar with the nature of our engineering team and the context it exists in.
+---
+
+## After your first week
+
+You’ve done general people-ops and engineering team onboarding, and have casually met some team-members. Now it’s time to start onboarding as a new engineering manager.
+
+- [ ] If you’re reading this, you’re the first external engineering manager we’ve hired and your name is Adam. Yay! (If not, this will be _real_ awkward.)
+
+     At Artsy we are trying to give new hires ownership over improving the onboarding documentation that they are faced with, in an effort to help the next new hire onboard more smoothly. As you are the first one, there is plenty of work to do in this area; in fact, this very document is nothing but a first draft. Things in this document may turn out to be insufficient, too broad, or simply not work at all. That’s all good and you should feel empowered to bring this up openly.
+
+     It will be your task to make this document into a substantially larger one, that draws from all the things you had to learn. You’re not on your own, your fellow peers will all be available to support you during this task. You got this!
```

Comment:
Maybe doesn’t have to be “substantially larger” but could definitely be “improved.”